src/NServiceBus.Core/EndpointCreator.cs

```diff
@@ -126,7 +126,9 @@ void Initialize()
                 eventAggregator,
                 criticalError,
                 settings.ErrorQueueAddress(),
-                hostingComponent);
+                hostingComponent,
+                pipelineSettings,
+                containerComponent.ContainerConfiguration);
```

Comment:
We usually pass the whole component but I see nothing wrong with just passing the container config 👍